mirror-wordpress-plugins

```diff
@@ -89,11 +89,11 @@ class PluginUpdater
     if !prefix.empty?
       @topics.each do |topic|
         if topic.start_with?("#{prefix}:")
-          RepositoryTopic.new(topic.split(":")[1], prefix).remove_from_repo(@repo)
+          RepositoryTopic.new(topic.split(":")[1], prefix).remove_from_repo()
```

Comment:
Should we be passing in the repo reference here?

Reply:
This ended up in [c16ad26](https://github.com/dxw-wordpress-plugins/mirror-gitlab-plugins/pull/24/commits/c16ad26841b32ef19834836c51a6aa83564c3741) and will need squashing